core/arch/arm/mm/tee_pager.c

```diff
@@ -910,11 +911,20 @@ static struct pgt **alloc_merged_pgt_array(struct vm_paged_region *a,
 {
 	size_t a_next_pgt_count = get_pgt_count(a_next->base, a_next->size);
 	size_t a_pgt_count = get_pgt_count(a->base, a->size);
+	size_t pgt_count = get_pgt_count(a->base, a->size + a_next->size);
```

Comment:
`_maybe_unused` for `pgt_count`

Reply:
Oh, I see that I forgot to replace one occurence of `get_pgt_count(a->base, a->size + a_next->size)`.